.travis.yml

```diff
@@ -9,39 +9,47 @@ matrix:
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
+        - USE_CGAL=1
+        - USE_VTK=1
     - os: osx
       compiler: clang
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
         - BLASLAPACK_IMPLEMENTATION=vecLib
+        - BUILD_DOCUMENTATION=1
     - os: osx
       compiler: clang
       env:
         - USE_PROJECT=0
-        - USE_SYSTEM=0
         - BLASLAPACK_IMPLEMENTATION=Auto
-    - os: osx
+    - os: linux
       compiler: clang
       env:
-        - USE_PROJECT=0
-        - BLASLAPACK_IMPLEMENTATION=OpenBLAS
```

Comment:
are you still checking OpenBLAS? I find it better to explicit each linalg backend for each built. It's easier to see what is tested